tensorstore/driver/ometiff/metadata.h

```diff
@@ -0,0 +1,134 @@
+#ifndef TENSORSTORE_DRIVER_OMETIFF_METADATA_H_
+#define TENSORSTORE_DRIVER_OMETIFF_METADATA_H_
+
+#include <string>
+#include <map>
+#include <tuple>
+
+#include "absl/status/status.h"
+#include <nlohmann/json.hpp>
+#include "tensorstore/array.h"
+#include "tensorstore/codec_spec.h"
+#include "tensorstore/data_type.h"
+#include "tensorstore/internal/json_binding/bindable.h"
+#include "tensorstore/schema.h"
+#include "tensorstore/serialization/fwd.h"
+#include "tensorstore/strided_layout.h"
+#include "tensorstore/util/garbage_collection/fwd.h"
+#include "tensorstore/util/result.h"
+#include "tensorstore/util/span.h"
+
+namespace tensorstore {
```

Comment:
All your utilities should live in this namespace.

Reply:
moved utilities in omexml.* under `tensorstore::omexml` namespace